components/locale_canonicalizer/Cargo.toml

```diff
@@ -33,6 +33,7 @@ all-features = true
 [dependencies]
 icu_locid = { version = "0.4", path = "../locid" }
 icu_provider = { version = "0.4", path = "../../provider/core", features = ["macros"] }
+litemap = { version = "0.2", path = "../../utils/litemap", features = ["serde_serialize"] }
```

Comment:
issue: The feature should not be enabled here, it should be enabled in `provider_serde` below as `"litemap/serde_serialize"`

Reply:
Done, but may I know what the difference is?